packages/app-project/stores/User/UserPersonalization/YourStats/YourStats.js

```diff
@@ -12,6 +12,22 @@ function statsHost(env) {
   }
 }
 
+export const statsClient = {
+  fetchDailyStats: async function fetchDailyStats({ projectId, userId }) {
```

Comment:
Maybe use [object method syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions) here. 

```suggestion
  async fetchDailyStats({ projectId, userId }) {
```